File: restaurant-crud-api/restaurant-crud-api/Data/DTO/Response/CustomerResponse.cs
using System.ComponentModel.DataAnnotations;

namespace restaurant_crud_api.Data.DTO.Response
{
    public class CustomerResponse
    {
        public int CustomerId { get; set; }

        public string CustomerName { get; set; }

        public string CustomerAddress { get; set; }

        public string CustomerPhone { get; set; }
    }
}

File: SimpleOnlineShop/online-shop-be/src/main/java/com/template/data/dto/response/CustomerResponse.java
package com.template.data.dto.response;

import java.time.LocalDate;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class CustomerResponse {
    private Integer customerId;
    private String customerCode;
    private String customerPic;
    private String customerName;
    private String customerPhone;
    private String customerAddress;
    private Boolean isActive;
    private LocalDate lastOrderDate;
}
